tests/cypress/integration/cmsWorkflow/unlockResubmit.spec.ts

```diff
@@ -91,8 +89,19 @@ describe('dashboard', () => {
 
                 cy.visit(reviewURL)
 
-                // state user can resubmit and see resubmitted package in dashboard
+                // State user can resubmit and see resubmitted package in dashboard
                 cy.intercept('POST', '*/graphql').as('gqlRequest2')
+
+                //Unlock banner for state user to be present with correct data.
+                cy.findByTestId('unlockedBanner')
+                    .should('exist')
+                    .and('contain.text', 'zuko@example.com')
+                    .and('contain.text', 'Unlock submission reason.')
+                    .contains(
+                        /Unlocked on: (0?[1-9]|[12][0-9]|3[01])\/[0-9]+\/[0-9]+\s[0-9]+:[0-9]+[a-zA-Z]+\s[a-zA-Z]+/i
```

Comment:
What is this regex? Are we just trying to match a mm/dd/yyyy timestamp?

Reply:
Yes, this regex is just checking the timestamp is in the correct format. I was having issues getting the correct current date time in cypress testing.

Reply:
I think so, I grabbed this from Jason's work. I think he was seeing the date bug here where the date was not correct  (off by one day I believe) so instead added regex which I thought was creative workaround. @JasonLin0991 am I describing the bug you saw right? Did it sound similar to this https://qmacbis.atlassian.net/browse/OY2-12629

Reply:
> I think he was seeing the date bug here where the date was not correct (off by one day I believe)

Just checking some past cypress tests done today and I think it's similar, except it seems like in cypress the time will always be `12:00am UTC`.

Reply:
oh I see it’s a whole timestamp, I thought we were just going to show the date here. I’m a little worried about the bug appearing here, I thought that was just with dates that had no time associated with them. Here we should be getting a bonafied Date() object which should do timezone adjustments automatically.